node-src/tasks/verify.ts

```diff
@@ -244,12 +242,13 @@ export const verifyBuild = async (ctx: Context, task: Task) => {
     }
   }
 
+  ctx.log.info(storybookPublished(ctx));
```

Comment:
@ghengeveld it looks like the addition of this on line 79 above was meant to print this all the time, so I moved it out of the conditional block down here since the build on the context at this point is in a correct state for logging component and story counts. But please let me know if I'm overlooking other reasons for it being added above.

Reply:
Yeah I think this is fine.